packages/sku-codemod/README.md

```diff
@@ -0,0 +1,31 @@
+[![npm](https://img.shields.io/npm/v/sku-codemod.svg?style=flat-square)](https://www.npmjs.com/package/sku-codemod)
+
+# Codemods for `sku`
+
+## Usage
+
+```sh
+pnpm dlx sku-codemod <codemod> [files] [options]
+```
```

Comment:
Should it also be mentioned that `pnpm dlx sku-codemod` run without `codemod` or `files` will show prompts for those options?